nobiru/nobiru_list.py

```diff
@@ -16,3 +16,12 @@ def delete(self, entity):
             db.session.commit()
         except ValueError:
             abort(404, f"{entity._name()} not found")
+
+    @collection.replaces(1)
+    def update(self, entity, new_entity):
+        try:
+            self.remove(entity)
+            self.append(new_entity)
+            db.session.commit()
+        except ValueError:
+            abort(404, f"{entity.name()} note found")
```

Comment:
Yeah, I did a dive into the sqlalchemy docs to try and find a straight update collections function like you could do with postgres, but I couldn't find one.  I was hoping that someone else might have encountered one.  

I think that because we are prescribing the note id in the resource, and then appending it straight into the db, that is keeping the id consistent.  If we used the NotesModel.create() method it would create a note id during that process.  

Should we keep looking for a better way to do this?